### DISCUSSION TITLE
How to write an object to generate 'oneof items'?

### DISCUSSION THREAD

**Discussion by ErikXu** [dotnet/dotnet-operator-sdk#409] [Q&A]
Hello, I want to know how to write an object to generate 'oneof items' like the istio does? eg:
https://github.com/istio/istio/blob/1.13.1/manifests/charts/base/crds/crd-all.gen.yaml#L3764

**Comment by buehler** [comment]
Hey @ErikXu 
Right now, this is not possible with the current state of KubeOps
